controllers/idle_handlers.go

```diff
@@ -18,17 +18,75 @@ package controllers
 
 import (
 	"context"
+	"fmt"
+	"os"
+	"path/filepath"
 
 	lcav1alpha1 "github.com/openshift-kni/lifecycle-agent/api/v1alpha1"
+	"github.com/openshift-kni/lifecycle-agent/controllers/utils"
+	ostree "github.com/openshift-kni/lifecycle-agent/ibu-imager/ostreeclient"
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+func cleanupUnbootedStateroot(ostreeClient *ostree.Client, stateroot string) error {
```

Comment:
Actually all this can be part of ostreeclient no?

Reply:
yes, this will be moved to the new module once it is developed.